api/doc/CHANGELOG.md

```diff
@@ -4,6 +4,39 @@ All notable changes to the Aptos Node API will be captured in this file. This ch
 
 **Note**: The Aptos Node API does not follow semantic version while we are in active development. Instead, breaking changes will be announced with each devnet cut. Once we launch our mainnet, the API will follow semantic versioning closely.
 
+## 1.1.0 (2022-09-08)
+- The `/events/{event_key}` endpoint has been removed. In addition to it no longer featuring in any request to the API, the canonical string representation of `EventKey` no longer appears in any response from the API either. Where previously you might see a representation of an event like `OLD` below, it now looks like `NEW`. See [#4012](https://github.com/aptos-labs/aptos-core/pull/4012).
+- A new endpoint has been added for getting events by GUID: `/accounts/{address}/events/{creation_number}`. This should be more easily usable by hand than the previous API.
```

Comment:
Given this is a new endpoint, this would amount to just keeping the old endpoint around for a couple of weeks. Under normal circumstances this is what I'd do, and we'd slowly deprecate the old endpoint, notify people lots, increasingly start throwing errors from the old endpoint, etc. So far we haven't really done that with the API though in the interest of moving fast. I'm happy with either approach though. Thoughts @gregnazario?

Reply:
Either way, for the SDK I'd just delete the old function and add the new one in the next version. Users can update to the new version when they're ready, vs the API in which they have no choice.

Reply:
`next version` == next major version?

Reply:
We should do a migration translation for them.